pubtools/pulplib/_impl/model/repository/base.py

```diff
@@ -35,6 +36,39 @@ def decorate(klass):
     return decorate
 
 
+@attr.s(kw_only=True, frozen=True)
+class Importer(PulpObject):
+    """
+    Importer is a pulp object that needs to be associated with repository
+    in order to successfully sync or upload content to it.
+    """
+
+    type_id = pulp_attrib(default=None, type=str, pulp_field="importer_type_id")
+    """
+    Type id of the importer.
+    """
+    config = pulp_attrib(default=None, type=dict, pulp_field="config")
```

Comment:
Could this please use frozendict (`converter=frozendict_or_none_converter`) so the object is immutable?